config/end-to-end.ini

```diff
@@ -11,22 +11,24 @@ case=extract_and_output
 extract_options=--no-vss
 output_file=extract_and_output1.log
 output_format=dynamic
-reference_output_file=test_data/end_to_end/extract_and_output1.log
+reference_output_file=test_data/end_to_end/extract_and_output.log
 source=test_data/vsstest.qcow2
 
 [extract_and_output2]
 case=extract_and_output
 extract_options=--no-vss --text C:
 output_file=extract_and_output2.log
 output_format=dynamic
-reference_output_file=test_data/end_to_end/extract_and_output2.log
+reference_output_file=test_data/end_to_end/extract_and_output_with_text_prepend.log
 source=test_data/vsstest.qcow2
 
 [extract_and_output3]
 case=extract_and_output_with_psteal
+extract_options=--no-vss
 output_file=extract_and_output3.log
```

Comment:
Please change this to match the reference_output_file name

Reply:
IMHO no, since that would cause the extract_and_output3 test to overwrite this file in the test results directory

test_data/end_to_end/extract_and_output.log is used for both the extract_and_output1 (psort) and extract_and_output3 (psteal) tests. Hence the description.